### PR TITLE
Add script to generate DAI for given testchain address

### PR DESCRIPTION
**Pull request by grandizzy**
usage: > node_modules/.bin/babel-node scripts/generateDai.js <address> <daiAmount>

### CODE REVIEW

**Comment by levity** [comment]
Maybe this script should live somewhere else? It seems to be very specific to some use case, e.g. it not only generates Dai for an address but sets the price and debt ceiling for the entire testchain.

File: scripts/generateDai.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+// usage:
+// > node_modules/.bin/babel-node scripts/fundTestAccount.js <address>
+
+import Maker from '../packages/dai/src';
```
Comment (levity) [inline scripts/generateDai.js]: please use absolute imports, i.e. `@makerdao/dai`; once you run the setup ([docs](https://github.com/makerdao/dai.js#getting-started)) they will use the local builds anyway

File: scripts/generateDai.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+// usage:
+// > node_modules/.bin/babel-node scripts/fundTestAccount.js <address>
```
[the remark this answers is not in the document]
Comment (grandizzy) [reply scripts/generateDai.js]: done

File: scripts/generateDai.js (hunk 1 above)
Comment (grandizzy) [reply scripts/generateDai.js]: done

**Comment by grandizzy** [comment]
> Maybe this script should live somewhere else? It seems to be very specific to some use case, e.g. it not only generates Dai for an address but sets the price and debt ceiling for the entire testchain and removes the allowance for the address.

Oops, I clean out script to leave only part for generating DAI

**Comment by levity** [comment]
> > Maybe this script should live somewhere else? It seems to be very specific to some use case, e.g. it not only generates Dai for an address but sets the price and debt ceiling for the entire testchain and removes the allowance for the address.
> 
> Oops, I clean out script to leave only part for generating DAI

Let's move this into the test-helpers package since it's only for the testchain. You can create `packages/test-helpers/scripts` and put it in there

**Comment by grandizzy** [comment]
> > > Maybe this script should live somewhere else? It seems to be very specific to some use case, e.g. it not only generates Dai for an address but sets the price and debt ceiling for the entire testchain and removes the allowance for the address.
> > 
> > 
> > Oops, I clean out script to leave only part for generating DAI
> 
> Let's move this into the test-helpers package since it's only for the testchain. You can create `packages/test-helpers/scripts` and put it in there

Done with last commit. Following this structure - shouldn't fundTestAccount.js be moved in same place as generateDai.js? Thanks

**Comment by levity** [comment]
Yes, quite right. I'll take care of that

**Pull request merged**